cmd/send.go

```diff
@@ -21,8 +22,8 @@ var sendCmd = &cobra.Command{
 	Short: "Send JUnit test reports to Fern",
 	Args:  cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
-		if err := client.SendReports(fernUrl, projectName, filePattern, tags, verbose); err != nil {
-			fmt.Fprintf(os.Stderr, "ERROR: %s\n", err)
+		if err := client.SendReports(util.RealClock{}, fernUrl, projectName, filePattern, tags, verbose); err != nil {
```

Comment:
any reason why we need to send an extra variable for RealClock? Meaning why could we not call this method as needed for time handling ?

Reply:
I'm not sure what your suggestion is here, how would we determine if SendReports, or any other function being tested, is called by a test or not?

Reply:
My question is why is being passed in. So is the purpose to help determine if it's a test or not ?

Reply:
ok thanks. It was more of trying to understand. You right it works and the test are passing.

Reply:
Moved to GlobalClock, no longer passed via args